src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2372,11 +2376,8 @@ private boolean isFeedbackSessionViewableTo(
         }
```

Comment:
The comment above can be rephrased. `//If the session is a private session created by the same user, it is viewable to the user`